iis2iclx_reg.c

```diff
@@ -6835,10 +6837,11 @@ int32_t iis2iclx_sh_slv0_cfg_read(const stmdev_ctx_t *ctx,
 
   if (ret == 0)
   {
-    slv0_add.slave0 = (uint8_t) val->slv_add >> 1;
+    slv0_add.slave0 = (uint8_t) val->slv_add;

```

Comment:
good catch, but put it in a different commit with proper commit message

Reply:
Done